sympy/physics/mechanics/system.py

```diff
@@ -0,0 +1,437 @@
+from sympy import eye, Matrix, zeros
+from sympy.physics.mechanics import dynamicsymbols
+from sympy.physics.mechanics.functions import find_dynamicsymbols
+
+__all__ = ['SymbolicSystem']
+
+
+class SymbolicSystem(object):
+    """SymbolicSystem is a class that contains all the information about a
+    system in a symbolic format such as the equations of motions and the bodies
+    and loads in the system.
+
+    There are three ways that the equations of motion can be described for
+    Symbolic System:
+
+
+        [1] Explicit form where the kinematics and dynamics are combined
+            x' = F_1(x, t, r, p)
+
+        [2] Implicit form where the kinematics and dynamics are combined
+            M(x, p) x' = F_2(x, t, r, p)
+
+        [3] Implicit form where the kinematics and dynamics are separate
+            M(q, p) u' = F_3(q, u, t, r, p)
+            q' = G(q, u, t, r, p)
+
+    where
+
+    x : states, i.e. [q, u]
+    t : time
+    r : specified (exogenous) inputs
+    p : constants
+    q : generalized coordinates
+    u : generalized speeds
+    M : mass matrix or generalized inertia matrix (implicit dynamical equations
+    or combined dynamics and kinematics)
+    F : right hand side (implicit dynamical equations or combined dynamics and
+    kinematics)
+    G : right hand side of the kinematical differential equations
```

Comment:
I'll change the indentation and see if the errors reappear. It might have just been in the .rst file it didn't like the indentation and this was the only way I could get rid of the error.